### DISCUSSION TITLE
Include a list of valid `enum` values if an invalid `enum` value is detected during config loading.

### DISCUSSION THREAD

**Discussion by jcollie** [ghostty-org/ghostty#2578] [Feature Requests, Ideas] [answered]
As discussed on Discord earlier, it would be very helpful to include a list of valid `enum` values in the error message if an invalid `enum` value is detected during config loading. For simplicity's sake it would only be added for `enum` values at this time but in theory this could be extended in the future to cover more types.

https://discord.com/channels/1005603569187160125/1005603569711452192/1302333119277437039

Here's a sample of what this might look like:

![image](https://github.com/user-attachments/assets/55b7a7d8-fee0-4792-ade1-5c317b9f5ef0)

**Comment by mitchellh** [comment, marked as answer]
Accepted #2635.
